Content.Server/Administration/WatchlistWebhookSystem.cs

```diff
@@ -0,0 +1,142 @@
+using Content.Server.Administration.Notes;
+using Content.Server.Database;
+using Content.Server.Discord;
+using Content.Shared.CCVar;
+using Robust.Server;
+using Robust.Server.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Network;
+using System.Linq;
+using Timer = Robust.Shared.Timing.Timer;
+
+namespace Content.Server.Administration;
+
+/// <summary>
+///     This system sends a Discord webhook notification whenever a player with an active
+///     watchlist joins the server.
+/// </summary>
+public sealed class WatchlistWebhookSystem : EntitySystem
```

Comment:
This is an out-of-simulation system, it should not use an entity system and should instead use a regular IoC manager.